src/rtcModule/webrtc.cpp

```diff
@@ -2885,8 +2894,15 @@ void Session::msgSessTerminate(RtMessage& packet)
     assert(packet.payload.dataSize() >= 1);
     cmd(RTCMD_SESS_TERMINATE_ACK);
 
+    TermCode code = static_cast<TermCode>(packet.payload.read<uint8_t>(8));
+
     if (mState == kStateTerminating)
     {
+        if (mTermCode == TermCode::kInvalid || (!isTermRetriable(code) && isTermRetriable(mTermCode)))
```

Comment:
If already terminating, there should be a termcode already set:
```suggestion
        assert(mTermCode != TermCode::kInvalid);
        if (!isTermRetriable(code) && isTermRetriable(mTermCode))
```